.neutrinorc.js

```diff
@@ -1,103 +1,37 @@
 module.exports = {
   use: [
-    ['./packages/airbnb-base', {
+    ['./packages/airbnb', {
```

Comment:
As mentioned in the other comment, we don't actually use any react in the repo, and aren't linting the create-project JSX files in this repo using our own configuration. In my opinion, I think this should stay as airbnb-base.

Reply:
If that's the case, we should remove the react eslint plugin references from `,neutrinorc.js` then - since we are already using a React based linting config - this change was keeping the status quo.

That said, I wonder if we should actually eventually remove the create-project templates from `.eslintignore` and start linting them like we do the rest of the repository. Yes we'll still need the one off lint step during create-project bootstrap, but at least we'd catch/fix more obvious issues in the code we have checked in - plus it would improve consistency with the rest of the monorepo from a code style/best practices perspective.

Reply:
> but at least we'd catch/fix more obvious issues in the code we have checked in

I think this is a compelling reason to at least lint it ourselves...somehow. It would be nice to catch eslint **errors** at patch-time instead of finding them in release. Stylistic fixes will still be done during create-project.

Reply:
I was hoping the eslint `overrides' feature would let us do this, however it seemed like it only works via the ESLint API, only via the CLI sadly.